wiki/Bot_account/pt-br.md

```diff
@@ -0,0 +1,62 @@
+---
+tags:
+  - bot accounts
+  - chat bot
+  - chat bots
+  - IRC bot
+  - IRC bots
+---
+
+# Conta de bot
+
+Uma **conta de bot** é um tipo especial de conta no osu! gerenciada por um serviço automatizado ao invés de uma pessoa real. No website, elas têm uma insígnia de grupo branca escrito `BOT`, e apenas a seção `eu!` de seu perfil é visível. Contas de bot são comumente usada para hospedar "chat bots" que oferecem várias funcionalidades para a comunidade pelos canais do lobby [multiplayer](/wiki/Multi) ou mensagens privadas.
+
+Contas de bot só podem ser criadas pelo [processo de aplicação](#creating-a-bot-account). A tentativa de criar uma conta de bot registrando normalmente é considerado "multi-accounting" e é contra as [regras da comunidade](/wiki/Rules#community-rules).
+
+## Benefícios de contas de bot
+
+A principal diferença entre uma conta pessoal e contas de bot estão nos limites que se aplicam ao enviar mensagens no [chat](/wiki/Chat_Console). Contas de bot têm limites mais altos para permitir que seus serviços interajam com mais usuários sem risco de ter a conta [silenciada](/wiki/Glossary#silence):
+
+- Contas pessoais podem enviar 10 mensagens a cada 5 segundos
+- Contas de bot podem enviar 300 mensagens a cada 60 segundos.
+
+Estes limites se aplicam apenas para mensagens privadas, `#multiplayer`, e `#spectator`. Contas de bot não são permitidas de enviar mensagens em outros canais.
+
+## Criando uma conta de bot
+
+Se você está pensando em começar um chat bot, por favor inicie utilizando ele na sua conta do osu!, e certifique-se de respeitar os limites de conta pessoal listados acima. Assim que seu serviço crescer, pode ser uma oportunidade para solicitar uma conta de bot para o time de suporte de conta.
```

Comment:
```suggestion
Se você está pensando em começar um chat bot, por favor inicie utilizando ele na sua conta do osu!, e certifique-se de respeitar os limites de conta pessoal listados acima. Assim que seu serviço crescer, pode ser uma oportunidade para solicitar uma conta de bot para a equipe de suporte de conta.
```
Just for consistency with other pt-br articles. If `equipe` is not deemed too good, we could change it to `time` in other pt-br articles if needed.